tests/unit/type/list.tcl

```diff
@@ -1475,12 +1467,12 @@ foreach {pop} {BLPOP BLMPOP_LEFT} {
     foreach {type large} [array get largevalue] {
         test "RPOPLPUSH base case - $type" {
             r del mylist1{t} mylist2{t}
-            create_list mylist1{t} "a $large c d"
+            create_$type mylist1{t} "a $large c d"
             assert_equal d [r rpoplpush mylist1{t} mylist2{t}]
             assert_equal c [r rpoplpush mylist1{t} mylist2{t}]
             assert_equal "a $large" [r lrange mylist1{t} 0 -1]
             assert_equal "c d" [r lrange mylist2{t} 0 -1]
-            assert_encoding quicklist mylist2{t}
+            assert_encoding listpack mylist2{t}
```

Comment:
we use `create_$type` but then assert that it's always a `listpack`?
why is that? because the rpoplpush doesn't move the `$large` item? maybe add another one to the list?

Reply:
Yes, added another one to the list. (https://github.com/redis/redis/pull/11303/commits/196e101dd6b8ae1340ee3feaaec46adb6c200144)